features/bevragen/persoon/gezag/gba/gezag-minderjarige-gba.feature

```diff
@@ -0,0 +1,179 @@
+      | 000000012       | G          |                  |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam | waarde             |
+      | type | TijdelijkGeenGezag |
+
+
+  Rule: wanneer een niet-ouder van rechtswege gezag heeft over de minderjarige wordt deze voogdij in gezag aangegeven 
+
+    Scenario: de partner van overleden ouder heeft gezag over de minderjarige
+      Gegeven voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | V          | 000000024        |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde    |
+      | type                             | Voogdij   |
+      | minderjarige.burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'derde' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+  Rule: wanneer van rechtswege gezag is toegewezen aan een voogd dan is de voogd niet bekend en wordt gezag geleverd zonder belanghebbenden
+
+    Scenario: er is een gerechtelijke uitspraak tot gezag voor ouder 1 en een derde
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'gezagsverhouding' gegevens
+      | naam                                 | waarde |
+      | indicatie gezag minderjarige (32.10) | 1D     |
+      En voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | V          |                  |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde    |
+      | type                             | Voogdij   |
+      | minderjarige.burgerservicenummer | 000000012 |
+      En heeft 'gezag' geen 'derde' gegevens
```

Comment:
Dus hiermee zeggen we dat derden een lege array is en niet één of twee voogden zonder gegevens. Maw:
```
"derden": []
```
en niet
```
"derden": [{}]
```
of
```
"derden": [{}, {}]
```

Reply:
klopt, zo had ik het ook bedoeld. we weten namelijk niks over de voogdij, alleen dat het er is.
Bij voogdij kunnen we meerdere situaties bestaan:
- er is 1 derde
- er zijn 2 derden
- er is derde en 1 ouder

Alleen de eerste situatie (1 derde) kan nu door de gezagsmodule worden geleverd, wanneer die derde van rechtswege gezag heeft. De andere situaties (gezag na rechtelijke uitspraak) kunnen niet uit de BRP worden afgeleid.

Reply:
In dit geval moet de stap worden veranderd naar `En heeft 'gezag' geen derden`. Anders kan er in de automation geen onderscheid worden gemaakt tussen een derde (=object) bij GezamenlijkGezag en derde (=array) bij Voogdij

Reply:
aangepast